requirements.txt

```diff
@@ -6,8 +6,10 @@ MySQL-python==1.2.3rc1
 python-dateutil==1.4.1
```

Comment:
I think it makes sense to also bump some other versions:
django==1.8.19
lxml==3.3.0
python-dateutil==2.4

Reply:
.....and done